src/systems/a320_systems_wasm/src/lib.rs

```diff
@@ -0,0 +1,117 @@
+#![cfg(any(target_arch = "wasm32", doc))]
+use a320_systems::A320;
+use msfs::{
+    legacy::{AircraftVariable, NamedVariable},
+    MSFSEvent,
+};
+use std::collections::HashMap;
+use systems::simulation::{Simulation, SimulatorReaderWriter};
+
+#[msfs::gauge(name=systems)]
+async fn systems(mut gauge: msfs::Gauge) -> Result<(), Box<dyn std::error::Error>> {
+    let mut simulation = Simulation::new(A320::new(), A320SimulatorReaderWriter::new()?);
+
+    while let Some(event) = gauge.next_event().await {
+        if let MSFSEvent::PreDraw(d) = event {
+            simulation.tick(d.delta_time());
+        }
+    }
```

Comment:
Any possibility around here to check if we are in FS2020 menu or ingame? Seems to me that WASM refreshes in menu as well.

Reply:
Thus far we have not found a way to do that. Other system modules (such as the fly by wire law simulation) also run into this. We hope this will be improved by Asobo.